contrib/opencensus-context/examples/py37.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+
+from opencensus.common.runtime_context import RuntimeContext
+
+RuntimeContext.register_slot('correlation_context', lambda: dict())
+
+async def hello(name):
+    correlation_context = RuntimeContext.correlation_context.copy()
+    correlation_context['name'] = name
+    RuntimeContext.correlation_context = correlation_context
```

Comment:
You wouldn't expect to copy and reassign using the actual correlation context package right?

Reply:
Right.

Reply:
The actual correlation context library will provide the interface for user to consume (e.g. it could be copy-on-write or immutable).